x-pack/plugins/stack_alerts/public/alert_types/threshold/validation.test.ts

```diff
@@ -32,7 +32,7 @@ describe('expression params validation', () => {
   });
   test('if aggField property is invalid should return proper error message', () => {
     const initialParams: IndexThresholdAlertParams = {
-      index: ['test'],
+      index: 'test',
```

Comment:
Changing this test so that we have a jest test that will test a `string` index param (instead of the usual `string[]` type).  It would be good to add a function test, I think, would need to create an alert using the API (passing a `string` typed `index` field), and then edit the alert and make sure the single index appears as expected in the UI.  Do we have similar tests like this?  Thoughts on where to add such a test?

Reply:
Could you use the `esArchiver` to load an archived alert with a string index param and then try to edit it in your functional test?

Reply:
I think we should be able to create one via the API - I suspect that will be easier to figuring out esArchiver :-)

Let me see if I can add a quick one ...

Reply:
I found one - I ended up changing the test in `x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/details.ts`, below, from using an array of indices to a single index string.